api/tests/Data/Tests/Core/ContainerConstantsTests.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RaceResults.Data.Core;
+
+namespace Internal.RaceResults.Data.Core
+{
+    [TestClass]
+    public class ContainerConstantsTests
+    {
+        [TestMethod]
+        public void ContainerNameTest()
+        {
+            Assert.AreEqual("MemberContainer", ContainerConstants.MemberContainerName);
```

Comment:
may want to also assert the # of constants defined is expected, so if someone adds one we know to update this file

Reply:
I don't think this is supper necessary. The main reason this is here is because the names need to match the deployed container names otherwise we'll get a runtime error. The number of containers is not really part of the implicit spec of this file.